### PR TITLE
CON-251 Update snapback / stateMachine Envvars

### PR DESCRIPTION
**Pull request by SidSethi**
Pairs with this PR https://github.com/AudiusProject/audius-protocol/pull/3716

- Remove `disableSnapback` envvar as it is no longer in use. It's been removed in this PR https://github.com/AudiusProject/audius-protocol/pull/3716, and won't affect nodes that don't have that code yet since it is defaulted to `disableSnapback = true`
- Remove `snapbackMaxLastSuccessfulRunDelayMs` envvar as it isn't used anywhere
- Rename `snapbackHighestReconfigMode` to `stateMachineReconfigMode` to go along with [this change in PR](https://github.com/AudiusProject/audius-protocol/pull/3716/files#diff-55efed0ab787151206047915ddb41d05c19d050dd660d6c15cffd55d801eae53R418). This is safe to do because in any race condition, nodes will use existing defaults which are safely set, and will disable reconfig modes
- Change `stateMachineReconfigMode` default for staging to `PRIMARY_AND_OR_SECONDARIES`

### CODE REVIEW

File: creator-node/prod.env [hunk 1]
```diff
@@ -23,7 +22,7 @@ logLevel=info
 maxAudioFileSizeBytes=1000000000
 pinAddCIDs=QmXSa6NxfA3e2hCpd8P5gNnZJs9PjZ3yGDHiRHR5B7Rq52,QmNU5Sv8se1aqfUFYFxupX1wdZfGTGnJp6vpoMbLNceGVi
 setTimeout=3600000
-snapbackHighestReconfigMode=RECONFIG_DISABLED
+stateMachineHighestReconfigMode=RECONFIG_DISABLED
```
Comment (dmanjunath) [inline creator-node/prod.env:25]: do we need this at all? like if the config inside the container has this we should just let that specify for 3p nodes

File: creator-node/stage.env [hunk 2]
```diff
@@ -24,8 +23,7 @@ logLevel=info
 maxAudioFileSizeBytes=1000000000
 pinAddCIDs=Qma5FGErNqHA32cyEjgp8hnwCFoqyYcBXU1ySKPA6MUJS5,QmYPy9A3zxLeBBuk1vpPNeG5aoMfa5qLNjJgh473Z9zV9b
 setTimeout=3600000
-snapbackHighestReconfigMode=RECONFIG_DISABLED
-snapbackMaxLastSuccessfulRunDelayMs=18000000 # 5hrs in ms
+stateMachineHighestReconfigMode=PRIMARY_AND_OR_SECONDARIES
```
Comment (dmanjunath) [inline creator-node/stage.env:26]: one note about this - this is actually as an override on stage nodes. we could clear it in one place. but also your PR description [here](https://github.com/AudiusProject/audius-protocol/pull/3716) is not entirely accurate but i don't think it'll cause any problems

File: creator-node/prod.env (hunk 1 above)
Comment (SidSethi) [reply creator-node/prod.env:25]: fair - i guess this was jsut for explicitness, but i'm down to remove

Comment (dmanjunath) [reply creator-node/prod.env:25]: i think we should remove layers of indirection to make rollout simpler. the two issues i can potentially see here is if we forget to update this  only update config.js then it won't roll out correctly or if a node operator copies the env vars by hand every now and then (which some do) and they don't update this on time

**Review by theoilie — APPROVED** [review]
not sure if this was blocked on me, but I don't see any big risks here

**Comment by SidSethi** [comment]
no longer needed -closing

**Pull request closed without merging**